### ISSUE TITLE
Magisk keeps stopping pop-up error when xiaomi mi 9 (re)boot

### ISSUE THREAD

**Issue by rosucin** [topjohnwu/Magisk#2799]
tldr: apparently not a big problem, just annoing message.

xiaomi mi 9 password encrypted and flashed with magisk 20.4 from twrp
manager hidden under another name, flagged the options for hide and systemless
except fxexplorer adaway and cerberusapp every other app including system ones are flagged in magisk hide
apparently safetynet annoing bank gugolpay don't even know that it's rooted

problem:
everytime I reboot the phone and unlock it I get this error for like 10 seconds that manager keeps stopping to work, if I want to report to mipenguins and if I go to the details I get:

`java.lang.IllegalStateException: SharedPreferences in credential encrypted storage are not available until after user is unlocked
	at android.app.ContextImpl.getSharedPreferences(ContextImpl.java:468)
	at android.app.ContextImpl.getSharedPreferences(ContextImpl.java:453)
	at android.content.ContextWrapper.getSharedPreferences(ContextWrapper.java:188)
	at a.l50.a(Unknown Source:12)
	at androidx.work.impl.utils.ForceStopRunnable.run(:6)
	at a.m50.run(Unknown Source:2)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:919)
`

for a while I thought it was due to unflagged "keep forced encryption" or something like that under advanced in the main screen, but it's already flagged and reflag itself even if I undo it. as for "recovery mode", I thought it would be good to keep it flagged but resets itself too. 

any ideea? thanks

**Comment by topjohnwu** [comment]
This has nothing to do with Magisk

**Comment by rosucin** [comment]
sorry can you explain better for newbie like me? thanks
